modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -139,7 +141,7 @@ spec:
 +
 [source,terminal]
 ----
-$ HOST=$(kubectl get ingress -n echoserver echoserver -o json | jq -r '.status.loadBalancer.ingress[0].hostname')
+$ HOST=$(oc get ingress -n echoserver echoserver --template='{{index .status.loadBalancer.ingress 0}}{{.hostname}}')
```

Comment:
The original `-o json | jq ...` is correct, if we want a replacement with `--template`, the it should be like `--template='{{(index .status.loadBalancer.ingress 0).hostname}}'`

Reply:
Addressed